### PR TITLE
Fix/issue 17

### PR DESCRIPTION
**Pull request by cedricdcc**
This pull request also contains all changes that are in https://github.com/vliz-be-opsci/py-trav-harv/pull/45 .
Best to review PR 45 first.

### CODE REVIEW

File: travharv/config_build.py [hunk 1]
```diff
@@ -247,7 +248,7 @@ def __init__(
         :rtype: TravHarvConfigBuilder
         """
         if configFolder is None:
-            configFolder = os.path.join(os.getcwd(), "config")
+            configFolder = Path.cwd() / "config"
```
Comment (marc-portier) [inline travharv/config_build.py:251]: snake-case?

File: travharv/config_build.py [hunk 2]
```diff
@@ -316,9 +317,9 @@ def _assert_valid_sparql_syntax(self, sparql_query):
 
     def _files_folder(self):
         return [
-            f
-            for f in os.listdir(self.config_files_folder)
-            if f.endswith((".yml", ".yaml"))
+            f.name
+            for f in Path(self.config_files_folder).iterdir()
```
Comment (marc-portier) [inline travharv/config_build.py:321]: does iterdir recurse into subfolders like glob?
if not, would that not be nice to have?

File: travharv/config_build.py [hunk 3]
```diff
@@ -316,9 +317,9 @@ def _assert_valid_sparql_syntax(self, sparql_query):
...
+            if f.name.endswith((".yml", ".yaml")) and f.is_file()
```
Comment (marc-portier) [inline travharv/config_build.py:322]:
```
if path.is_file() and path.suffix in {". yml", ".yaml"} 
```

**Review by marc-portier — CHANGES_REQUESTED** [review]
minor / optional things in this one -- the actual needed changes are related to the ones commented in #45

File: travharv/config_build.py (hunk 2 above)
Comment (cedricdcc) [reply travharv/config_build.py]: added rglob("*") that does do recursive calling

**Pull request merged**